config/firefox-panel.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This is probably the most interesting piece of the PR. 

Previously there was development.json and local.json. Adding firefox-panel.json, lets us have separate configs for the panel or other environments we might support in the future.